gtsfm/two_view_estimator.py

```diff
@@ -218,16 +240,83 @@ def check_for_degeneracy(
         return i2Ri1, i2Ui1, v_corr_idxs, two_view_report
 
 
+def ground_truth_verifier(
+    keypoints_i1: Keypoints,
+    keypoints_i2: Keypoints,
+    matches_i1i2: np.ndarray,
+    intrinsics_i1: Cal3Bundler,
+    intrinsics_i2: Cal3Bundler,
+    dist_threshold: float,
+    gt_wTi1: Pose3,
+    gt_wTi2: Pose3,
+    gt_scene_mesh: trimesh.Trimesh,
+) -> Tuple[Optional[Rot3], Optional[Unit3], np.ndarray, float]:
```

Comment:
Should this be a child class of `BaseVerifier`?

Reply:
I can definitely add it, as it's useful for debugging. What do you think?

Reply:
+1, even if its not a derived verifier (which sounds like a good idea), it would be nice to have it outside this file.